docs/codeql/codeql-language-guides/analyzing-data-flow-in-cpp-new.rst

```diff
@@ -0,0 +1,431 @@
+.. _analyzing-data-flow-in-cpp-new:
+
+.. pull-quote:: Note
+
+   The data flow library described here is available from CodeQL 2.13.0 onwards. See :ref:`here <analyzing-data-flow-in-cpp>` for the library available in earlier versions.
```

Comment:
The version number chosen here is clearly just random.

Reply:
Sounds like something ChatGPT would do. :P

Reply:
It'll likely be 2.12.5. I'll update accordingly.

Reply:
Updated.